tests/framework/db/SchemaTest.php

```diff
@@ -161,7 +161,7 @@ public function getExpectedColumns()
                 'defaultValue' => 1,
             ],
             'char_col' => [
-                'type' => 'string',
+                'type' => 'char',
```

Comment:
This is causing the test to fail. Shouldn't the type be recognized as `'char'` now? If so, then something else is wrong.